### PR TITLE
Add missing properties of User

### PR DESCRIPTION
**Pull request by mvolpato**
I added some properties that were missing in the `User` model.

I also added a `Plan` model, which can be returned when you retrieve a user.
As far as I know there is no direct API to request plans.

Tests added for `Plan` and on the unit test on the `user_me.json` file.

### CODE REVIEW

**Review by MaxDesiatov — APPROVED** [review]
Seems legit

File: Tests/OctoKitTests/UserTests.swift [hunk 1]
```diff
@@ -95,6 +95,30 @@ class UserTests: XCTestCase {
...
+        XCTAssertEqual(subject.followingURL, "https://api.github.com/users/pietbrauer/following{/other_user}")
+        XCTAssertEqual(subject.gistsURL, "https://api.github.com/users/pietbrauer/gists{/gist_id}")
+        XCTAssertEqual(subject.starredURL, "https://api.github.com/users/pietbrauer/starred{/owner}{/repo}")
+        XCTAssertEqual(subject.subscriptionsURL, "https://api.github.com/users/pietbrauer/subscriptions")
+        XCTAssertEqual(subject.reposURL, "https://api.github.com/users/pietbrauer/repos")
+        XCTAssertEqual(subject.eventsURL, "https://api.github.com/users/pietbrauer/events{/privacy}")
+        XCTAssertEqual(subject.receivedEventsURL, "https://api.github.com/users/pietbrauer/received_events")
+        XCTAssertFalse(subject.siteAdmin!)
+        XCTAssertTrue(subject.hireable!)
+        XCTAssertNil(subject.bio)
```
Comment (pietbrauer) [inline Tests/OctoKitTests/UserTests.swift:110]: Currently I am not sure if this is mapped correctly or if the field is empty in the JSON. Could you change the stub and add a `bio`, `twitterUsername` and `noteId`?

File: OctoKit/User.swift [hunk 2]
```diff
@@ -19,6 +19,31 @@ open class User: Codable {
     open var numberOfPublicRepos: Int?
     open var numberOfPublicGists: Int?
     open var numberOfPrivateRepos: Int?
+    open var nodeId: String?
```
Comment (pietbrauer) [inline OctoKit/User.swift:22]: Could we also uppercase `Id`?

```suggestion
    open var nodeID: String?
```

**Review by pietbrauer — CHANGES_REQUESTED** [review]
Thanks a lot! Just some very minor things.

File: Tests/OctoKitTests/Fixtures/user_me.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "location": "Hamburg",
     "email": null,
     "hireable": true,
-    "bio": null,
+    "bio": "Tweeting about iOS and Drumming",
```
Comment (pietbrauer) [inline Tests/OctoKitTests/Fixtures/user_me.json:25]: Haha very humbled, wasn't aware it's my user in the stubs 😆

**Review by pietbrauer — APPROVED** [review]

**Comment by pietbrauer** [comment]
Thanks again for your contribution!

**Pull request merged**